internal/tiltfile/tiltfile_test.go

```diff
@@ -5879,6 +5882,7 @@ type fixture struct {
 	k8sContext k8s.KubeContext
 	k8sEnv     k8s.Env
 	webHost    model.WebHost
+	ctrlclient ctrlclient.Client
```

Comment:
Is this intentional (maybe going to be imminently used)? AFAICT the TiltfileLoader doesn't actually need the client (yet?)

Reply:
ya, i have a branch that unifies the reconciler with the existing extension system so you can do:

extension_repo(name='default', url='file://path/to/my/repo')
extension(name='kubefwd', repo_name='default', path='kubefwd')
load('ext://my-ext', ...)

and the tiltfile loader will ask the reconciler to load the extension instead. i think that will be a lot more flexible in general.